packages/inputs/src/features/options.ts

```diff
@@ -15,12 +15,23 @@ export type FormKitOptionsList = Array<
 >
 
 /**
- * Accepts an array of objects, array of strings, or object of key-value pairs.
- * and returns an array of objects with value and label properties.
+ * Options to be normalized
+ */
+type FormKitOptionsToBeNormalized = string[] | FormKitOptionsList | { [value: string]: string }
```

Comment:
@justin-schroeder I added this typing to get rid of `any` in the type definition for `options` in the `normalizeOptions` function — could use a gut check on if it's the right definition. The linter is happy with it.